script.rb

```diff
@@ -422864,7 +422864,7 @@ def raw_apply(snr)
 s.ins 0x4f, :addr_0x930a8, [-1]
 s.ins 0x4f, :addr_0x92d9a, [9, 362, 0, -9]
 s.ins 0x8e, 0, 0, 10, byte(0)
-s.ins 0x86, ushort(1441), byte(1), byte(1), s.layout('Jessica@r@v04/JES_00030."B-Battler, you arsehole, what do you think you\'re saying to my mom?!"')
+s.ins 0x86, ushort(1441), byte(1), byte(1), s.layout('Jessica@r@v04/JES_00030."B-Battler, you asshole, what do you think you\'re saying to my mom?!"')
```

Comment:
This is intentional and was meant to represent Jessica's overly harsh manner of speech, just like other similar instances of this kind of thing throughout the script (it's also meant to poke fun at the fact that Jessica is British in the console version for no reason, but that's a more minor thing). Please revert this.